portfolyo/tools/duration.py

```diff
@@ -14,7 +14,7 @@ def stamp(ts: pd.Timestamp, freq: str) -> tools_unit.Q_:
     ----------
     ts : pd.Timestamp
         Timestamp for which to calculate the duration.
-    freq : {{{', '.join(tools_freq.FREQUENCIES)}}}
+    freq : {{{tools_freq.ALLOWED_FREQUENCIES_DOCS}}}
```

Comment:
I think we only need single (not tripple) braces

Reply:
done